### PR TITLE
W-14338712-associate-environments-kt

### PR DESCRIPTION
**Pull request by kevintroller**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: modules/ROOT/pages/associate-environments.adoc [hunk 1]
```diff
@@ -5,15 +5,66 @@ endif::[]
...
+== Associate Business Groups
+
+You can share Runtime Fabric with an organization's business groups. This allows other business groups to deploy applications to a single Runtime Fabric. To use this feature, the business groups must fall under the scope of the organization the Runtime Fabric is registered to.
+
+To associate a Runtime Fabric across business groups:
+
+. From Anypoint Platform, select *Runtime Manager*.
+. Select the *Runtime Fabric* tab on the left sidebar.
+. Select the Runtime Fabric to manage.
+. On the *Environments* tab, click *Add business group*.
```
Comment (hegderohit89) [inline modules/ROOT/pages/associate-environments.adoc:20]: Correction:
On the Associated Environments tab, click Add business group.

File: modules/ROOT/pages/associate-environments.adoc [hunk 2]
```diff
@@ -5,15 +5,66 @@ endif::[]
...
+Enter the business group name in the *Add business group* field and select it from the list. This option allows you to select specific environments for specific child business groups.
+
+[start=6]
+. Click *Save Changes* or *Discard Changes* to discard or confirm the association.
+
+=== Add or Remove Business Groups
+
+You can add or remove business groups:
+
+To add new business groups, use the *Add business group* field.
```
Comment (hegderohit89) [inline modules/ROOT/pages/associate-environments.adoc]: I dont think this is right. We can remove this `=== Add or Remove Business Groups` section al together. 
The first section already covered adding BGs and Child orgs

@aishwarya-murali what do you think ?

File: modules/ROOT/pages/associate-environments.adoc (hunk 1 above)
Comment (kevintroller) [reply modules/ROOT/pages/associate-environments.adoc:20]: Based on a screenshot provided by the team, the UI tab shows as Environments

File: modules/ROOT/pages/associate-environments.adoc [hunk 3]
```diff
@@ -5,15 +5,60 @@ endif::[]
...
+. On the *Environments* tab, click *Add business group*.
+. Select the business groups from the drop-down list:
+
+** *This business group and its children*
++ 
+This option allows you to select either all environments, all sandbox environment or all production environments for the selected business group and all its children.
+
+** *Selected business groups* 
++
+Enter the business group name in the *Add business group* field and select it from the list. This option allows you to select specific environments for specific child business groups.
```
Comment (aishwarya-murali) [inline modules/ROOT/pages/associate-environments.adoc]: `select specific environments for specific child business groups` -> You can select specific environments for root org as well as it's children. So, can we change this to `specific business groups`

**Comment by aishwarya-murali** [comment]
[@kevintroller](https://github.com/kevintroller) For RTF, we should mention that Appliance/Gravity type RTF will not support this feature. This type of RTF will only support older environment association.

**Review by aishwarya-murali — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/associate-environments.adoc [hunk 4]
```diff
@@ -5,15 +5,60 @@ endif::[]
...
 . Select the *Runtime Fabric* section on the left sidebar.
 . Select the Runtime Fabric based on the name used during installation.
-. From the *Associated Environments* tab, select the environment you want to associate with this Runtime Fabric.
-. Click *Apply* to confirm the changes.
+. Click the *Environments* tab.
+. Select the environments from the drop-down list:
 
-== Production Environments
+** *All Environment* +
+Associates the private space with all sandbox and production environments in the business group and all its children, or in specific child business organizations.
```
Comment (hpanchal1221) [inline modules/ROOT/pages/associate-environments.adoc]: It should be `Associates the Runtime Fabric` for this all all subsequent statements.

**Comment by hpanchal1221** [comment]
Could we add screenshots for RTF as well like we have added for CH2?

**Comment by aishwarya-murali** [comment]
@kevintroller This comment above - https://github.com/mulesoft/docs-runtime-fabric/pull/1157#discussion_r1376241264 should be applied for all cases (I see them in `Select the environments from the drop-down list:` sections).

File: modules/ROOT/pages/associate-environments.adoc (hunk 3 above)
Comment (kevintroller) [reply modules/ROOT/pages/associate-environments.adoc]: Changed to:

_This option allows you to select specific environments for root organizations as well as it's children._

Let me know if this is clear. Also, please indicate which other sentences need to be updated with this clarification.

**Review by luanamulesoft — APPROVED** [review]

**Review by aishwarya-murali — APPROVED** [review]

**Pull request merged**